PairedSingleSampleWf_noqc_nocram_optimized.wdl

```diff
@@ -476,8 +482,8 @@ task MarkDuplicates {
       ${"READ_NAME_REGEX=" + read_name_regex} \
       OPTICAL_DUPLICATE_PIXEL_DISTANCE=2500 \
       ASSUME_SORT_ORDER="queryname" \
-      CLEAR_DT="false" \
-      ADD_PG_TAG_TO_READS=false
+      #CLEAR_DT="false" \

```

Comment:
Why were these parameters commented out? In latest version of this workflow these tools still uses the parameters [link](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline/blob/d78f3886fde1a7146f0318552cb2a335eac70dde/tasks/BamProcessing.wdl#L134).

Reply:
These changes were something we inherited. We can revert and test again.

Reply:
A person from Davids team can better determine whether these parameters are necessary, I just wanted to point out the difference and provide a link to the Broads current pipeline.